MediaGalleryUi/view/adminhtml/web/js/directory/directoryTree.js

```diff
@@ -316,6 +316,8 @@ define([
                 $(this.directoryTreeSelector).jstree('deselect_all');
                 this.activeNode(null);
                 this.directories().setInActive();
+            } else {
+                this.checkChipFiltersState();
```

Comment:
I think it would be good to:
 - rename the function `checkChipFiltersState` to `updateSelectedDirectory`
 - update listens configuration: `'${ $.provider }:params.filters.path': 'updateSelectedDirectory'`
 - keep only the logic of clearing filters selection in `clearFiltersHandle` function (move the condition to the `updateSelectedDirectory`)